src/material/checkbox/checkbox.scss

```diff
@@ -167,13 +196,27 @@
   // user of this. Therefore we add the pointer cursor on top of MDC's styles.
   label {
     cursor: pointer;
+
+    @include token-utils.use-tokens(
+      tokens-mat-checkbox.$prefix,
+      tokens-mat-checkbox.get-token-slots()
+    ) {
+      @include token-utils.create-token-slot(color, label-text-color);
```

Comment:
These styles aren't super complicated, but should we move them into a common mixin so we don't have to repeat any bugfixes?

Reply:
I'm not really sure its worth the extra layers of indirection, I'd rather just be slightly repetitive.